Source/Core/DolphinWX/Config/GeneralConfigPane.cpp

```diff
@@ -198,7 +199,8 @@ void GeneralConfigPane::OnDualCoreCheckBoxChanged(wxCommandEvent& event)
   if (Core::IsRunning())
     return;
 
-  SConfig::GetInstance().bCPUThread = m_dual_core_checkbox->IsChecked();
+  Config::SetBase(Config::CPU_THREAD, m_dual_core_checkbox->IsChecked());
+  Config::InvokeConfigChangedCallbacks();
```

Comment:
Perhaps it should be called in Config::Set, that way every single configuration change would trigger callbacks, though this *would* complicate batching if one wanted to implement it. :p

I'm personally not too worried about performance since configuration seems to only change at start-up and when a user changes it from the UI, but I'll defer to your better judgement.